dotCMS/src/main/java/com/dotcms/integritycheckers/IntegrityUtil.java

```diff
@@ -614,7 +780,8 @@ public void completeDiscardConflicts(final String endpointId) throws DotDataExce
      * @return is there is at least one conflict returns true, otherwise false
      * @throws Exception
      */
-    public boolean completeCheckIntegrity(final String endpointId) throws Exception {
+    @WrapInTransaction
+    public static boolean completeCheckIntegrity(final String endpointId) throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6151243)